pages/checkout/ideal.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div>
+    <div id="payment-page">
+      <div class="container">
+        <div class="payment-container">
+          <div id="component-container" class="payment"></div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+
+import { ref, onMounted } from 'vue';
+import { AdyenCheckout, Redirect } from '@adyen/adyen-web';
+
+// Reactive references
+const sessionId = ref('');
+const redirectResult = ref('');
+
+// Function to invoke server-side API
+async function sendPostRequest(url, data) {
+  const res = await fetch(url, {
+    method: "POST",
+    body: data ? JSON.stringify(data) : "",
+    headers: {
+      "Content-Type": "application/json",
+    },
+  });
+  return await res.json();
+}
+
+// Function to handle the server response
+async function handleServerResponse(res, component) {
+  if (res.action) {
+    component.handleAction(res.action);
```

Comment:
Q: Do we need this check for iDeal's redirect?

Reply:
Yes, iDEAL has become a `Redirect` payment method and uses the action to redirect to the iDEAL Hub page